Create Handbook section (#800)
mattermost/mattermost-handbook: operations/operations/handbook/handbook-onboarding.md

# Getting Started with the Handbook

The Mattermost Handbook is a work-in-progress in that information is constantly being updated, added, revised, and edited.

## Editing the Handbook

The Handbook is open to the public and edits can be submitted by anyone.

## Adding content

> You don't need to ask permission to add content to the Handbook. Follow the steps in "How do I edit the Handbook" to get started.

Reviewer: A direct link here would be ideal

> You don't need to ask permission to add content to the Handbook. Follow the steps in "How do I edit the Handbook" to get started.

Author: ```suggestion
You don't need to ask permission to add content to the Handbook. Follow the steps in [How to update the Handbook](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-update-handbook) to get started.
```
